### DISCUSSION TITLE
smartdns和本地代理会重复吗？

### DISCUSSION THREAD

**Discussion by taifeng12** [pymumu/smartdns#1810] [General]
问题一：在路由器上安装了smartdns，pc上开启了clash或V2rayN工具，这时候路由器上的smartdns是否多余？
问题二：smartdns46的上游服务器页，上游服务器地址和加入ipset地址可填写不同的国外ip吗？不明白这一页的设置具体目的

**Comment by PikuZheng** [comment]
1. 不一定，取决于clash配置的dns服务器是什么
2. (我的luci没这项)如果不知道该怎么填，请保持空白
